lib/routes/withdraw/reverse_swap/confirmation_page/widgets/reverse_swap_button.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/bloc/withdraw/withdraw_funds_bloc.dart';
+import 'package:c_breez/utils/exceptions.dart';
+import 'package:c_breez/widgets/error_dialog.dart';
+import 'package:c_breez/widgets/loader.dart';
+import 'package:c_breez/widgets/single_button_bottom_bar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class ReverseSwapButton extends StatelessWidget {
+  final int amountSat;
+  final String onchainRecipientAddress;
+  final String feesHash;
+  final int satPerVbyte;
+
+  const ReverseSwapButton({
+    Key? key,
+    required this.amountSat,
+    required this.onchainRecipientAddress,
+    required this.feesHash,
+    required this.satPerVbyte,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final texts = context.texts();
+
+    return SingleButtonBottomBar(
+      text: texts.sweep_all_coins_action_confirm,
+      onPressed: () => _sendOnchain(context),
+    );
+  }
+
+  Future _sendOnchain(BuildContext context) async {
+    final texts = context.texts();
+    final withdrawFundsBloc = context.read<WithdrawFundsBloc>();
+
+    final navigator = Navigator.of(context);
+    var loaderRoute = createLoaderRoute(context);
```

Comment:
Do we want to use the easyloading instead? Was it better from development experience?

Reply:
It's great when you want to display loaders without context, don't think it's needed here.
It has it's benefits and I haven't checked it internal implementation but it's a bit alarming that it was last updated 16 months ago.

You can declare a global spinner instance instead of calling createLoaderRoute() as we do here and even override it when calling show().
It requires less typing and worrying about context management, but more mental load in terms of management if you don't want your app to get stuck with a spinner if you don't allow dismiss on tap.

It's a great plugin but we haven't encountered any issues that requires contextless spinners and also it not being maintained for so long is worrisome. Let's not add it unless absolutely necessary.